### PR TITLE
Upgrade react native from 0.65.3 to 0.70.2

### PR DESCRIPTION
**Pull request by tnagorra**

### CODE REVIEW

File: android/app/src/main/jni/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13)
+
+# Define the library name here.
+project(rndiffapp_appmodules)
```
Comment (laurentS) [inline android/app/src/main/jni/CMakeLists.txt:4]: same comment as above

**Review by laurentS — COMMENTED** [review]
I just took a quick look through the PR, in case something obvious would pop out.
I'm surprised by how many new `.h|.cpp` files are in the codebase, and also that there's a committed `gradle-wrapper.jar` binary file in the repo.
I picked up a couple of incorrect references to `rndiff` which is the default name in the RN upgrade diff tool, maybe that explains why nothing is starting? Hopefully this is helpful!

File: android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      return new MainApplicationTurboModuleManagerDelegate(context, packages);
+    }
+  }
+
+  @Override
+  protected synchronized void maybeLoadOtherSoLibraries() {
+    if (!sIsSoLibraryLoaded) {
+      // If you change the name of your application .so file in the Android.mk file,
+      // make sure you update the name here as well.
+      SoLoader.loadLibrary("rndiffapp_appmodules");
```
[the remark this answers is not in the document]
Comment (tnagorra) [reply android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java:44]: Very interesting find @laurentS 
I grepped the whole project to identify where we have `rndiffapp` and interestingly we already had `package com.rndiffapp;` in line 7 of `android/app/src/debug/java/org/missingmaps/mapswipe/ReactNativeFlipper.java`

Do you know if it has been used or why it does not crash?

[the remark this answers is not in the document]
Comment (laurentS) [reply android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java:44]: That was clearly my mistake :) My guess is that flipper is a debugger, and we/I never used it, so the code you mention was probably never used?

[the remark this answers is not in the document]
Comment (tnagorra) [reply android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java:44]: That is my guess as well. We will try renaming rndiffapp to mapswipe and see if it resolves the problem.

The weird thing is that the development release has no problem. It is the production release that crashes.

[the remark this answers is not in the document]
Comment (laurentS) [reply android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java:44]: Strange indeed. Is that when you run the dev release as an apk the same way you run prod, or do you run the dev release via the debugger? In that latter case, I think the app initialisation code is slightly different, which might explain why you're seeing different behaviours.

[the remark this answers is not in the document]
Comment (tnagorra) [reply android/app/src/main/java/org/missingmaps/mapswipe/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java:44]: That might be it.

**Pull request closed without merging**